### DISCUSSION TITLE
scoop update error: ERROR System.Net.Sockets.SocketException (0x80004005)

### DISCUSSION THREAD

**Discussion by scil** [ScoopInstaller/Scoop#4888] [General]
Then I tried `iwr -useb get.scoop.sh` or `iwr -useb  baidu.com`  , both saied `iwr 无法连接到远程服务器`( iwr can not connect remote server).

At last I install another powershell from microsoft store, everything works well.

Just share to be helpful.

**Comment by rashil2000** [comment]
Might be a problem with Geoblocking or GFW... which was resolved in PowerShell Core.

**Reply by scil** [reply]
> Might be a problem with Geoblocking or GFW... which was resolved in PowerShell Core.

Thank you. I tried to bypass GFW, not work.   even 'baidu.com' can not be accessed
